taigit/src/libraries/GitHub/src/avgComments.ts

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+/**
+ * The following function returns the Average Number Of Comments 
+ * on all closed Pull Requests by considering direct comments on
+ * pull requests and review comments.
+ */
```

Comment:
This isn't proper javadoc format. If you'd like a reference here's one! 
https://wiki.cs.jmu.edu/student/java/javadoc